org.swi_prolog.swipl.yml

```diff
@@ -0,0 +1,40 @@
+id: org.swi_prolog.swipl
+runtime: org.kde.Platform
+runtime-version: "6.7"
+sdk: org.kde.Sdk
+command: swipl-win
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=host
+  - --device=dri
+
+modules:  
+  - name: swipl
+    sources:
+      - type: git
+        url: https://github.com/SWI-Prolog/swipl-devel.git
+        tag: V9.3.7
+        commit: 93ec6b480bacec44134ae7e48e9759dc49a5ec4a
+        x-checker-data:
+          type: git
+          tag-pattern: "^V(\\d\\.[\\d.]+)$"
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=PGO
+      - -DSKIP_SSL_TESTS=ON
+  - name: launcher
+    buildsystem: simple
+    build-commands:
+      - install -D -m0644 org.swi_prolog.swipl.png /app/share/icons/hicolor/128x128/apps/org.swi_prolog.swipl.png
```

Comment:
this is not the actual size of the icon.

Reply:
Will change